tasks/console-errors.js

```diff
@@ -0,0 +1,85 @@
+const puppeteer = require('puppeteer');
+const { getSitemapUrls } = require('./get-sitemap-links');
+
+const excludedErrors = [
+  {
+    type: 'Shortbread',
+    errorText:
+      "Shortbread failed to set user's cookie preference because the domain name that was passed"
```

Comment:
Can we add a comment on why the shortbread script is expected to fail (for when we are wondering later)